sql/analysis/deviations_v1/query.sql

```diff
@@ -0,0 +1,10 @@
+SELECT
+  date,
+  ci_deviation,
+  deviation,
+  geography,
+  metric
+FROM
+  `moz-fx-data-derived-datasets.analysis.deviations`
```

Comment:
@jklukas I saw this bug about [decomissioning derived datasets](https://bugzilla.mozilla.org/show_bug.cgi?id=1612543). Does that mean that this data should rather be written somewhere else?

Reply:
Yes, we should write it to `moz-fx-data-shared-prod.analysis.deviations`